### ISSUE TITLE
Replicas ip_configuration - enable_private_path_for_google_cloud_services for replicas

### ISSUE THREAD

**Issue by amais33** [terraform-google-modules/terraform-google-sql-db#468]
### TL;DR

The parameter "enable_private_path_for_google_cloud_services" under " ip_configuration" for replicas is not enabled in google module. Hence we are unable to enable it from terraform. When we enable it from console, terraform is overwriting the value to null. This parameter is available in terraform resource, but not available in the google module. When we try to add this parameter in lifecycle ignore changes, it is not accepting. Error is "The block type name "lifecycle" is reserved for use by Terraform in a future version".
This should be the same work as https://github.com/terraform-google-modules/terraform-google-sql-db/issues/442

### Terraform Resources

```markdown
resource "google_sql_database_instance" "instance" {
  provider = google-beta

  name             = "private-instance-${random_id.db_name_suffix.hex}"
  region           = "us-central1"
  database_version = "POSTGRES_14"

  depends_on = [google_service_networking_connection.private_vpc_connection]

  settings {
    tier = "db-f1-micro"
    ip_configuration {
      ipv4_enabled                                  = false
      private_network                               = google_compute_network.private_network.id
    }
  }

  read_replicas = [
      name                  = "private-instance-${random_id.db_name_suffix.hex}-replica",
      tier                  = "db-f1-micro"
      availability_type     = "ZONAL"
      zone                  = "us-central1"
      disk_type             = "PD_SSD",
      disk_autoresize       = true,
      disk_autoresize_limit = 0,
      disk_size             = 10,
      ip_configuration = {
        ipv4_enabled        = false
        private_network     = google_compute_network.private_network.id
        enable_private_path_for_google_cloud_services = true
      },
      encryption_key_name = null
  ]
}
```

### Detailed design

```markdown
Terraform plan when private path enabled from console

 ~ resource "google_sql_database_instance" "replicas" {
        id                             = "mydb-replica"
        name                           = "mydb-replica"
        # (15 unchanged attributes hidden)

      ~ settings {
            # (12 unchanged attributes hidden)

          ~ ip_configuration {
              - enable_private_path_for_google_cloud_services = true -> null
                # (3 unchanged attributes hidden)
            }

            # (16 unchanged blocks hidden)
        }

        # (2 unchanged blocks hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```
